tests/test_store.py

```diff
@@ -92,16 +94,25 @@ def test_get_id_returns_404_if_not_stored(self):
     def test_get_valid_id_returns_id_and_200(self):
         test_json = json.loads(updated_message)
         expected_id = test_json['tx_id']
+        response_size_original = sys.getsizeof(json.dumps(test_json))
+        response_hash_original = hashlib.md5(json.dumps(test_json, sort_keys=True).encode('utf-8')).hexdigest()
```

Comment:
if you pass `updated_message` into these lines then you don't need to do `json.dumps()`?

Reply:
I agree with that, but then how would the keys be sorted?

Reply:
is `updated_message` not in the correct order?

Reply:
It could be as we just have that written in test_data, but do we know that will always be the same order? Is it not better to sort and make sure?